src/lib.rs

```diff
@@ -2,6 +2,8 @@ extern crate libc;
 
 mod ffi;
 
+pub use ffi::rill_col;
```

Comment:
How does specifying a column work? Something like `rill::rill_col::rill_col_a` If so there's a few too many `rill` in there and I'd try to figure something that's a bit less verbose.